test/EndToEnd.Tests/GivenUnixPlatform.cs

```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace EndToEnd.Tests
+{
+    public class GivenUnixPlatform(ITestOutputHelper log) : SdkTest(log)
+    {
+        [UnixOnlyTheory]
```

Comment:
This previously had:
> [UnixOnlyTheory(Skip="https://github.com/dotnet/templating/issues/1979")]

If it is still broken, I'll have to add it back. But I didn't test it locally after converting it.